01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,187 @@ using InteractiveUtils
+Technology is the practice that allows us as humans to transform reality. 
+And here it is important to clarify that it is not something "new”, it has been with us since the very beginning of the human era, and it is really what makes us different from other animals. 
+
+Technology enables us to expand our capabilities.
+From taking a long, pointy stick to reach fruit high up in a tree, through transforming an entire landscape by building a huge electric dam, to the creation of artificial intelligence that helps us solve our deepest puzzles. 
+With technology we transform our experience in the world.
+
+It is evident that the more we know about the reality in which we live, the more we will be able to modify it. 
+But from this idea comes a very important question: what is the order in which innovation takes place? 
+Is it necessary first to have an absolute understanding of the process we are trying to modify? 
+Or is technology created in a more chaotic process, through trial and error?
+
+So let's think for a moment, does a child need to know the gear mechanisms of a bicycle in order to be able to use it? Does Lionel Messi need to know about fluid dynamics to make the ball take a curved trajectory? Did the Romans need to know the Navier-Stokes equation in order to build their huge aqueducts? 
+Maybe you are thinking that these are very specific cases, but the reality is that those examples share a general and universal property: the knowledge is acquired based on experimentation, implementation and heuristics, pure trial and error and with much less theoretical knowledge than many believe. 
+And technological innovation tends to follows the same mechanism, it was discovered by people that were taking risks, taking action, with a pure trial-and-error experiential manner and, above all, without anyone truly understanding the theory behind.
+
+The current understanding of how knowledge is produced does not match how it actually happens, and may even work against our society's development.
+Nowadays, there is a kind of linear thinking model in which it is taken for granted that the process of knowledge production has a very specific way of occurring: first, academic and theoretical discoveries are made. And only then, with all of that formalization achieve, one can start thinking of putting it into practical use.
+This type of thinking is counterproductive because, being so widespread, it causes many people to be more concerned with constantly acquiring theoretical knowledge, rather than taking action and immersing themselves in practice.
+
+By looking at the history of technology and innovation a repeating pattern can be seen: there is a constant differentiation between the people who do and the people who watch and then narrate. 
```

Comment:
By looking at the history of technology and innovation, and who writes it, we see that the people who made the discoveries are rarely the ones who write the books about them.